docs/feeds.md

```diff
@@ -0,0 +1,106 @@
+
+<!-- Our cloudant changes feed is connection based.  Put a link here to
+an open repo -->
+
+# Difference between Feed and Trigger
+
+Feeds and triggers are closely related,
+but technically distinct concepts.   
+
+- OpenWhisk processes **events** which flow into the system.
+
+- A **trigger** is technically a name for a class of events.   Each event belongs to exactly one trigger; by analogy, a trigger resembles a *topic* in topic-based pub-sub
+systems.    A **rule** *T -> A* means "whenever an event from trigger *T* arrives, invoke action *A* with the trigger payload.
+
+- A  **feed** is a stream of events which all belong to some trigger *T*.    A feed is controlled by a **feed action** which handles creating, deleting, pausing, and resuming the stream of events which comprise a feed.    The feed action typically interacts with external services which produce the events, via a REST API that manages notifications.
+
+#  Implementing Feed Actions
+
+The *feed action* is a normal OpenWhisk *action*, but it should accept the following parameters:
+* **lifecycleEvent**: one of 'CREATE', 'DELETE', 'PAUSE', or 'UNPAUSE'
+* **triggerName**: the fully-qualified name of the trigger which contains events produced from this feed.
+* **authKey**: the Basic auth credentials of the OpenWhisk user who owns the trigger just mentioned
+
+The feed action can also accept any other parameters it needs to manage the feed.  For example the cloudant changes feed action expects to receive parameters including *'dbname'*, *'username'*, etc.
+
+When the user creates a trigger from tthe CLI with the **--feed** parameter, the system automatically invokes the feed action with the appropriate parameters.
+
+For example,assume the user has created a `mycloudant` binding for the `cloudant`
+package with their username and password as bound parameters.  When the user issues the following command from the CLI,
+
+`wsk trigger create T --feed mycloudant/changes -p dbName myTable`
+
+then under the covers the system will do something equivalent to:
+
+`wsk action invoke mycloudant/changes -p lifecycleEvent CREATE -p triggerName T -p authKey <userAuthKey> -p password <password value from mycloudant binding> -p username <username value from mycloudant binding> -p dbName mytype`
+
+The feed action named *changes* takes these parameters, and is expected to take whatever action is necessary to set up a stream of events from Cloudant, with the appropriate configuration, directed to the trigger *T*.    
+
+For the Cloudant *changes* feed, the action happens to talk directly to a *cloudant trigger* service we've implemented with a connection-based architecture.   We'll discuss the other architectures below.
+
+A similar feed action protocol occurs for `wsk trigger delete`.    
+
+We have not yet implemented pause and unpause, but they will be similar.
+
+# Implementing Feeds with Hooks
```

Comment:
Yeah I was also on the fence, because it can be anything. But I think using the term Webhook helps with SEO and page search. and for "web" it means http based hooks and most common way that people refer to.